nifi-nar-bundles/nifi-standard-services/nifi-record-serialization-services-bundle/nifi-record-serialization-services/pom.xml

```diff
@@ -86,12 +86,23 @@
             <artifactId>avro</artifactId>
             <version>1.8.1</version>
         </dependency>
+        <dependency>
```

Comment:
I think the more appropriate dependency to use here is
```
<dependency>
    <groupId>net.java.dev.stax-utils</groupId>
    <artifactId>stax-utils</artifactId>
    <version>20070216</version>
</dependency>
```
No?

Either way, both have a dependency on BEA's JSR173, which has the following license: https://www.calculate-linux.org/packages/licenses/bea.ri.jsr173 - I'm not entirely sure if this is permissible in Apache land and if it is, how to properly document it. That said, it appears that jsr 173 is now part of java and so that dependency I think can be excluded. I changed the pom as follows:

```
<dependency>
            <groupId>net.java.dev.stax-utils</groupId>
            <artifactId>stax-utils</artifactId>
            <version>20070216</version>
            <exclusions>
                <exclusion>
                    <groupId>com.bea.xml</groupId>
                    <artifactId>jsr173-ri</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```
And all appears to work just fine. Thoughts?